### PR TITLE
Thx page

### PR DESCRIPTION
**Pull request by FStanczyk**
Visually finished v1.

To do:
 - share button handling
 - address submitting handling
 - opensea.io link
 - routing by hash(?)

### CODE REVIEW

**Comment by FStanczyk** [comment]
ToFix: 
- inconsistency with design: on mobile resolution, "Thank you {name}" should be at the top of ThxNode showcase.
- inconsistency with design: no underlines

**Comment by FStanczyk** [comment]
Added CrashCourse page and Collection Page.

ToDo:
   - finish thx-node carousel on collection page
   - routing

**Comment by FStanczyk** [comment]
commit #0031000:
NFT's on Collection page and on Thx-page are now taken right from IPFS links. 
Two debug buttons added to make the thx-page and collection page accessible for debugging.

toFix:
- logic issue: NodeCarousel has static number of pages (6 nodes on each) because the app does not know how many nft's needs to render. Easiest way would be to implement a "totalSupply" function in smart contract. Need to find good solution in web-app.
- design issue: NFT info's in collection page are visiible only after hover, showing the info's whole time resulted in div width distortions. Must make the info visible whole time, current implementation might not work with thx-nodes.

**Comment by FStanczyk** [comment]
ToFix:
- In nft-config.json it should be possible to add any number of attributes (in high numbers it might destroy design), now there are only 3.

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,13 @@
+FROM node:latest AS build
+WORKDIR /app
+COPY . /app
+
+RUN npm install
+RUN npm run build
+
+FROM ubuntu
```
Comment (kchudy) [inline Dockerfile]: Tutaj skorzystaj z obrazu nginx zamiast ubuntu

```
FROM node:19-alpine

WORKDIR /tmp/build

COPY package.json package.json
COPY yarn.lock yarn.lock
RUN yarn install

COPY public public
COPY src src
COPY tsconfig.json tsconfig.json
COPY config-overrides.js config-overrides.js
RUN yarn build

FROM nginx:1.21-alpine
COPY nginx.conf /etc/nginx/conf.d/default.conf
COPY --from=0 /tmp/build/build /usr/share/nginx/html
```

**Comment by FStanczyk** [comment]
> * logic issue: NodeCarousel has static number of pages (6 nodes on each) because the app does not know how many nft's needs to render. Easiest way would be to implement a "totalSupply" function in smart contract. Need to find good solution in web-app.

ERC721Psi has a function totalSupply

**Pull request merged**